### ISSUE TITLE
Change default handling of undefined annotations when selecting `most_severe_consequence`  in `modify_scout_vcf.pl`

### ISSUE THREAD

**Issue by alkc** [Clinical-Genomics-Lund/nextflow_wgs#149]
## TLDR

Missing annotations as added by new version of VEP are sorted out as most severe by perl.

The default behavior should be to rank them lowest by default or the script should crash

## Background

Consequences are ranked here:

https://github.com/Clinical-Genomics-Lund/nextflow_wgs/blob/b9eff369ae87bbd5f38829b9f8b57cf969ea24b5/bin/modify_vcf_scout.pl#L30-L68

and  retrieved/sorted here:

https://github.com/Clinical-Genomics-Lund/nextflow_wgs/blob/b9eff369ae87bbd5f38829b9f8b57cf969ea24b5/bin/modify_vcf_scout.pl#L223-L231

The value returned by the hash whe naccessing an undefined key is treated as a `0` in the sorting above, meaning that undefined annotations will always be ranked ahead of all other defined annotations, which all have values `> 0`

The script should crash to alert the dev that the new/missing/wrong annotations need to be handled. 

Furthermore, the ranking should be moved out of the script into some config space.
